integration-tests/src/integrationTest/java/pl/allegro/tech/hermes/integrationtests/management/TopicManagementTest.java

```diff
@@ -572,6 +573,40 @@ public void shouldNotUpdateTopicWithDisallowedLabels() {
         );
     }
 
+    @Test
+    public void shouldCreateTopicEvenIfExistsInBrokers() {
+        // given
+        String groupName = "existingTopicFromExternalBroker";
+        String topicName = "topic";
+        String qualifiedTopicName = groupName + "." + topicName;
+        hermes.initHelper().createGroup(Group.from(groupName));
+
+        brokerOperations.createTopic(qualifiedTopicName);
+
+        // when
+        WebTestClient.ResponseSpec response = hermes.api().createTopic((topicWithSchema(topic(groupName, topicName).build())));
+
+        // then
+        response.expectStatus().isCreated();
+        hermes.api().getTopicResponse(qualifiedTopicName).expectStatus().isOk();
+    }
+
+    @Test
+    public void topicCreationRollbackShouldNotDeleteTopicOnBroker() {
```

Comment:
1. Do you have an idea what is a difference between these two tests? I don't understand the where is the `Rollback` part coming from. Perhaps these two test should be merged? 

2. Previously this test was using multi DC setup, do you think this was not relevant?